### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Larry850806**
This is my Facebook bot.
It use mongodb for response database.

### CODE REVIEW

**Comment by Schmavery** [comment]
Dude
https://github.com/Larry850806/facebook-chat-bot/blob/master/index.js#L6

**Comment by Larry850806** [comment]
The user and password is just for testing.
I have removed them in the code and Readme.
I also rewrite the Readme in detail.

**Comment by avikj** [comment]
@Larry850806  just letting you know that your password+username can be found in the commit history.

**Comment by bsansouci** [comment]
@Larry850806 https://github.com/Larry850806/facebook-chat-bot/blob/f1dc0d9aaa44dc2e5a762961865d18cb1b015f9f/index.js
The easiest way would be to delete the repository, delete the local `.git` folder and recreate the github/local repo.

**Comment by Schmavery** [comment]
Honestly the easiest move is just to change the password...

**Comment by Larry850806** [comment]
Thanks.
I have delete the repository and recreate the github repo.

**Comment by AllanWang** [comment]
I just stumbled across this readme. Didn't want to create another request, but I have made a module that goes on top of this one to create some features (mostly using firebase as well)

https://github.com/AllanWang/AllanBot-Public

New users can experiment with this as it makes it really easy to add features (plus the code is there as an example).

**Comment by logicx24** [comment]
I've also made a facebook bot that adds a bunch of slack convenience features to facebook messenger, along with a few other things. Adding functions is also incredibly simple: all it requires is making a new file that defines two methods and adding it to a folder, so it's a good starting point for beginners.

https://github.com/logicx24/MarkovsInequality

**Comment by bsansouci** [comment]
@Larry850806 We moved the projects at the bottom of the Readme as there were too many of them. Can you update this PR and we'll merge.

**Comment by Larry850806** [comment]
Thanks, I have moved the projects at the bottom of Readme.

File: README.md [hunk 1]
```diff
@@ -2,10 +2,7 @@
 The Official Facebook Chat API uses XMPP and is deprecated as of April 30th 2015. This is a non-official API that doesn't use XMPP.
 As of right now, the only way to automate the chat functionalities is to emulate the browser. This means doing the exact same GET/POST requests and tricking Facebook into thinking we're accessing the website normally. Because we're doing it this way, this API won't work with an auth token but requires the credentials of a Facebook account.
 
-Cool projects using facebook-chat-api:
-- [Kassy](https://github.com/mrkno/Kassy) - Kassy is a modular, easily extensible general purpose chat bot
-- [Marc Zuckerbot](https://github.com/bsansouci/marc-zuckerbot) - Facebook chat bot
-- [Marc Thuckerbot](https://github.com/bsansouci/lisp-bot) - Programmable lisp bot
+
```
Comment (bsansouci) [inline README.md]: Can you pull locally and resolve the merge conflicts? Make sure to just add yours at the end of the list at the bottom

**Comment by Larry850806** [comment]
Thanks, I have pull from Schmavery/master and just add my repo at the end of the list.

**Comment by bsansouci** [comment]
Cool. Thanks!

**Pull request merged**